### DISCUSSION TITLE
Welcome to Introduction-to-Image-Processing Discussions!

### DISCUSSION THREAD

**Discussion by ritwikraha** [ritwikraha/Introduction-to-Image-Processing#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with others passionate about the topic of image processing. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other members.
  * Welcome others and are open-minded💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ariG23498** [comment]
Thanks for opening up this space!
Would be lovely to tackle the ideas with the help of a great community.

**Reply by ritwikraha** [reply]
Sure thing!
